sdk/cosmos/azure-cosmos/azure/cosmos/_execution_context/aio/execution_dispatcher.py

```diff
@@ -126,6 +126,22 @@ async def _create_pipelined_execution_context(self, query_execution_info):
                                                                                         self._options,
                                                                                         query_execution_info)
             await execution_context_aggregator._configure_partition_ranges()
+        elif query_execution_info.has_hybrid_search_query_info():
+            hybrid_search_query_info = query_execution_info._query_execution_info['hybridSearchQueryInfo']
+            if not hybrid_search_query_info['take']:
```

Comment:
Shouldn't these have similar logic as the vector search above to get the top set?